models/user.js

```diff
@@ -89,6 +89,18 @@ UserSchema.methods.hashPassword = function(password) {
 };
 
 /**
+ * Create instance method for changing a password 
+ */
+ UserSchema.methods.changePassword = function(newPassword) {
+ 	if (newPassword && newPassword.length > 6) {
+ 		this.password = newPassword;
+ 		this.save();
```

Comment:
When the object gets saved, the validator (validateLocalStrategyPassword) is called and validates the string length is not empty and is long enough. You can imagine in the future if we decided to change the password length to something else, it would be a pain if the length is validated in twenty different places. This changePassword method should basically only set a new password